autogpt/prompt.py

```diff
@@ -82,6 +82,7 @@ def get_prompt() -> str:
         ),
         ("Execute Python File", "execute_python_file", {"file": "<file>"}),
         ("Generate Image", "generate_image", {"prompt": "<prompt>"}),
+        ("email","send_email",{"email":"<email>","subject":"<subject>","body":"<body of the email(string)>"})
```

Comment:
This needs to be edited to 

("email", "send_email", {"email": "<email>", "subject": "<subject>", "body": "<body of the email(string)>"}),

to correct

flake8 autogpt/ tests/ --select E303,W293,W291,W292,E305,E231,E302

autogpt/prompt.py:87:17: E231 missing whitespace after ','
autogpt/prompt.py:87:30: E231 missing whitespace after ','
autogpt/prompt.py:87:39: E231 missing whitespace after ':'
autogpt/prompt.py:87:49: E231 missing whitespace after ','
autogpt/prompt.py:87:59: E231 missing whitespace after ':'
autogpt/prompt.py:87:71: E231 missing whitespace after ','
autogpt/prompt.py:87:78: E231 missing whitespace after ':'